src/index.ts

```diff
@@ -365,10 +374,13 @@ class Spanner extends GrpcService {
         ['A configuration object is required to create an instance.'].join('')
       );
     }
-    const formattedName = Instance.formatName_(this.projectId, name);
+    const formattedName = Instance.formatName_(
```

Comment:
Can we keep the `formatName_` functions written the same as before, and still just provide "this.projectId" as the first argument, instead of changing it to expect the "projects/" prefix?